hub/views/explore.py

```diff
@@ -114,7 +114,11 @@ def render_to_response(self, context, **response_kwargs):
                 ds = {**ds, **type_map[d.name]}
             if d.is_range:
                 ds["types"] = [
-                    {"name": dt.name, "title": dt.label}
+                    {
+                        "name": dt.name,
+                        "title": dt.label,
+                        "area_type": dt.area_type.code if dt.area_type else None,
+                    }
                     for dt in DataType.objects.filter(data_set=d)
```

Comment:
this should have a `.select_related("area_type")` otherwise the `dt.area_type.code` will be generating a SELECT for each loop entry.